MigrationCodelab/app/src/main/java/com/google/samples/apps/sunflower/plantdetail/PlantDetailFragment.kt

```diff
@@ -108,9 +109,22 @@ class PlantDetailFragment : Fragment() {
                 }
             }
 
-            composeView.setContent {
-                MdcTheme {
-                    PlantDetailDescription(plantDetailViewModel)
+            composeView.apply {
+                // By default, the Composition is disposed when ComposeView is detached
+                // from the Window. This default behavior is undesirable for fragments that
+                // can be created multiple times within the same Window. To dispose
+                // the Composition when the Fragment's LifecycleOwner is destroyed, we set the
```

Comment:
Wait, this isn't the Fragment's lifecycle owner being destroyed, it's the **View's** lifecycle being destroyed—right?